src/modules/reservationCounter.js

```diff
@@ -0,0 +1,16 @@
+const reservationCounter = (data) => {
+  let commentHtml = '';
+  let counter = 0;
+  if (data) {
+    data.forEach((d) => {
+      commentHtml += `<p>${d.date_start} - ${d.date_end} by ${d.username}</p>`;
+      counter += 1;
+    });
+
+    return counter;
+  }
+
+  return false;
+};
+
```

Comment:
- [ ] Again, let's ensure to indicate the reservation counter value on the reservation modal, like seen in the [template design](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md)

**Example of reservation counter value on reservation modal section:**
![image](https://user-images.githubusercontent.com/12745474/234876221-151f5d9f-00e2-4732-a03a-b2da91852ff4.png)

**Your comments reservation modal:**
![image](https://user-images.githubusercontent.com/12745474/234876676-3ef9cddc-488a-4408-a5ff-0152aaac73b5.png))